avalon/maya/commands.py

```diff
@@ -19,7 +19,12 @@ def reset_frame_range():
 
     try:
         edit_in = shot["data"]["edit_in"]
```

Comment:
maybe getting edit_in/out should be with `.get( )`

Reply:
Sure, that makes sense.